sql/updates/world/2018_04_18_02_world.sql

```diff
@@ -0,0 +1,5 @@
+-- Quest 28102 - Fight the power / Add Shadowstalkers to kill count
+UPDATE creature_template SET KillCredit1=7106 WHERE entry=7109;
+
```

Comment:
the quest wants "Slay 8 Jadefire Rogues (7106) or Jadefire Shadowstalkers (7110)"
the quest npcrequired is 7106 * 8
so the killkredit is 
"UPDATE creature_template SET KillCredit1=7106 WHERE entry=7110;"

Reply:
Yes you are right, i put wrong id (Jadefire Felsworn) by accident.